.github/workflows/main.yml

```diff
@@ -75,6 +75,11 @@ jobs:
       run: |
         sudo apt-get install build-essential swig python3-dev libedit-dev libncurses5-dev libxml2-dev
       if: matrix.os == 'ubuntu-22.04'
+    - name: Free Disk Space (Ubuntu)
```

Comment:
?

Reply:
The CI failed in the first run, because the Ubuntu runner ran out of storage:

> [Set up job](https://github.com/anza-xyz/platform-tools/pull/102#annotation:1:52)
You are running out of disk space. The runner will stop working when the machine runs out of disk space. Free space left: 70 MB

> The hosted runner: GitHub Actions 5 lost communication with the server. Anything in your workflow that terminates the runner process, starves it for CPU/Memory, or blocks its network access can cause this error.